rfcs/0167-maintainer-account.md

```diff
@@ -0,0 +1,260 @@
+
+### Upsides
+
+- nixpkgs does not depend on GitHub's policies and practices to determine who is a maintainer. This could be mitigated by migrating away from GitHub, but until that happens this point remains.
+- maintainers can remain more anonymous
+
+### Downsides
+
+- Maintainers without an account cannot be mentioned or referenced on the nixpkgs hosting platform
+- Users and maintainers must leave the nixpkgs hosting platform in order to contact a maintainers, creating fractured communications
+- Automation must account for users who do not have an account on the nixpkgs hosting platform
+- Another maintainer/contributor/committer must execute all actions on behalf of a maintainer who does not have an account
+- Automation may need to be written and maintained to accommodate contacting users without an account on the nixpkgs hosting platform
+
+## Move to another platform that is hosted by the community
+
+This is a non-trivial amount of effort and is out of the scope of this RFC.
+While this could alleviate some issues, it would not change the existing policy which allows maintainers to opt out of having an account on the nixpkgs hosting platform.
+
+# Prior art
+
+[prior-art]: #prior-art
+
+- [Linux Kernel Feature and driver maintainers](https://docs.kernel.org/maintainer/feature-and-driver-maintainers.html)
+- [Debian Maintainer](https://wiki.debian.org/DebianMaintainer)
+- [Kubernetes Community Membership](https://github.com/kubernetes/community/blob/master/community-membership.md)
+- [Fedora Joining the Package Maintainers](https://docs.fedoraproject.org/en-US/package-maintainers/Joining_the_Package_Maintainers/)
+- [openSUSE:How to contribute to Factory](https://en.opensuse.org/openSUSE:How_to_contribute_to_Factory)
+
+All of the prior art communities have a requirement for joining or participating in the hosting platform of the project.
+Some of these require an account for _all_ contributions, not just maintenance.
+
+- Kernel: `Maintainers must be subscribed and follow the appropriate subsystem-wide mailing list`
+- Debian: `register for a Salsa account if you do not have one`
+- Kubernetes: Does not explicitly state a GitHub account is required, but by reading the requirements which mention GitHub 2FA it seems clear that it is
+- Fedora: `Create a Bugzilla Account` and `Create a Fedora Account`
+- OpenSUSE: `To begin to contribute to factory, you will need to make an account on openSUSE.`
+
+# Unresolved questions
+
+[unresolved]: #unresolved-questions
+
+Is this touching on areas of law that justifies involving legal counsel?
+
+Would implementing a Developer Certificate of Origin (DCO) as a contract or a more explicit grant of personal data provide better understanding for maintainers?
+This could be added for signoff of modifications to `mainters-list.nix` only if it does provide the intendend behavior of granting nixpkgs permission to use data.
```

Comment:
```suggestion
This could be added for signoff of modifications to `maintainer-list.nix` only if it does provide the intendend behavior of granting nixpkgs permission to use data.
```